test/python/visualization/test_dag_drawer.py

```diff
@@ -35,7 +36,7 @@ def setUp(self):
 
     def test_dag_drawer_no_graphviz(self):
         """Test dag draw with no graphviz."""
-        with unittest.mock.patch('nxpd.pydot.find_graphviz', return_value=None) as _:
+        with patch('nxpd.pydot.find_graphviz', return_value=None) as _:
```

Comment:
the changes to this file seem unrelated.

Reply:
solved in 35be0787